File: Python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, id, nama_produk, kategori_produk, harga, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama_produk, kategori_produk, harga, jenis, bahan, warna)
        self.Untuk = untuk
        self.Size = size
        self.Merk = merk

    # Setter
    def setUntuk(self, untuk):
        self.Untuk = untuk
    def setSize(self, size):
        self.Size = size
    def setMerk(self, merk):
        self.Merk = merk

    # Getter
    def getUntuk(self):
        return self.Untuk
    def getSize(self):
        return self.Size
    def getMerk(self):
        return self.Merk

    # Fungsi utk menambahkan item
    def addBaju(self, id, nama_produk, kategori_produk, harga, jenis, bahan, warna, untuk, size, merk):
        self.set_ID(id)
        self.set_Nama_Produk(nama_produk)
        self.set_Kategori_Produk(kategori_produk)
        self.set_Harga(harga)
        self.set_Jenis(jenis)
        self.set_Bahan(bahan)
        self.set_Warna(warna)
        self.set_Untuk(untuk)
        self.set_Size(size)
        self.set_Merk(merk)

File: Java/Baju.java
class Baju extends Aksesoris {
    private String Untuk;
    private String Size;
    private String Merk;

    public Baju(int id, String nama_produk, String kategori_produk, int harga, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, nama_produk, kategori_produk, harga, jenis, bahan, warna);
        this.Untuk = untuk;
        this.Size = size;
        this.Merk = merk;
    }

	// Setter
    void setUntuk(String untuk){this.Untuk = untuk;}
    void setSize(String size){this.Size = size;}
    void setMerk(String merk){this.Merk = merk;}

	// Getter
    String getUntuk(){return Untuk;}
    String getSize(){return Size;}
    String getMerk(){return Merk;}

    public void addBaju(int id, String namaProduk, String kategoriProduk, int harga, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        setId(id);
        setNamaProduk(namaProduk);
        setKategoriProduk(kategoriProduk);
        setHarga(harga);
        setJenis(jenis);
        setBahan(bahan);
        setWarna(warna);
        setUntuk(untuk);
        setSize(size);
        setMerk(merk);
    }
}